kartothek/io/eager_cube.py

```diff
@@ -445,14 +542,32 @@ def copy_cube(cube, src_store, tgt_store, overwrite=False, datasets=None):
         src_store, tgt_store, cube.ktk_dataset_uuid(cube.seed_dataset)
     )
 
+    # get the keys to copy for the cube
     keys = get_copy_keys(
         cube=cube,
         src_store=src_store,
         tgt_store=tgt_store,
         overwrite=overwrite,
         datasets=datasets,
     )
```

Comment:
Can getting the keys be part of the if clause?

Reply:
No: The original keys are needed in both cases, copy without rename (if clause) and copy with rename (else clause).

Reply:
Args of course. You are right, keep it.